zeppelin-plugins/launcher/yarn/src/main/java/org/apache/zeppelin/interpreter/launcher/YarnRemoteInterpreterProcess.java

```diff
@@ -316,6 +347,25 @@ private ContainerLaunchContext setUpAMLaunchContext() throws IOException {
     return amContainer;
   }
 
+  private URI resolveURI(String path) {
```

Comment:
It would be good if we could add tests with high coverage for this function, as the process is a bit special

Reply:
Addressed